site/content/how-to/traffic-management/redirects-and-rewrites.md

```diff
@@ -222,6 +244,285 @@ Server name: coffee-6db967495b-twn6x
 URI: /prices?test=v1&test=v2
 ```
 
+---
+
+## RequestRedirect example
+
+This example demonstrates how to redirect the traffic to a new location for the tea and soda applications, using `RequestRedirect` filters.
+
+---
+
+### Setup
+
+Create the **tea** and **soda** application in Kubernetes by copying and pasting the following block into your terminal:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tea
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tea
+  template:
+    metadata:
+      labels:
+        app: tea
+    spec:
+      containers:
+      - name: tea
+        image: nginxdemos/nginx-hello:plain-text
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tea
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: tea
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: soda
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: soda
+  template:
+    metadata:
+      labels:
+        app: soda
+    spec:
+      containers:
+      - name: soda
+        image: nginxdemos/nginx-hello:plain-text
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: soda
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: soda
+EOF
+```
+
+This will create the **tea** and **soda** services and deployments. Run the following command to verify the resources were created:
+
+```shell
+kubectl get pods,svc
+```
+
+Your output should include the **tea**, **soda** pod and the **tea**, **soda** service:
+
+```text
+NAME                          READY   STATUS      RESTARTS   AGE
+pod/soda-7c76d95586-dnc4n     1/1     Running   0          89m
+pod/tea-7b7d6c947d-s8djx      1/1     Running   0          120m
+
+
+NAME                 TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+service/soda         ClusterIP   10.96.230.208   <none>        80/TCP    89m
+service/tea          ClusterIP   10.96.151.194   <none>        80/TCP    120m
+```
+
+---
+
+### Configure a path redirect
+
+We will define two HTTPRoutes for the **tea** application: `tea`, which specifies the destination to handle redirected requests, and `tea-redirect` that redirect requests as follows:
+
+- `http://cafe.example.com/tea` to `http://cafe.example.com/organic`
+- `http://cafe.example.com/tea/origin` to `http://cafe.example.com/organic/origin`
+
+We will also define two HTTPRoutes defined for the **soda** application: `soda`, which specifies the destination to handle redirected requests, and `soda-redirect` that redirect requests as follows:
+
+- `http://cafe.example.com/soda` to `http://cafe.example.com/flavors`
+- `http://cafe.example.com/soda/pepsi` to `http://cafe.example.com/flavors`
```

Comment:
I think it would be helpful to point out which type of path redirect we are configuring here before showing the YAML.